### DISCUSSION TITLE
ciphey not working, maybe bad installation

### DISCUSSION THREAD

**Discussion by Tropical-Smootie** [bee-san/Ciphey#582] [Show and tell]
i don't think i installed it right and this is the response im getting, can someone help me troubleshoot this pls
im also using a macOS  #
<img width="779" alt="Screen Shot 2021-02-02 at 11 01 41 PM" src="https://user-images.githubusercontent.com/67304179/106710531-effb4680-65aa-11eb-9af1-68d76e3c1c66.png">

**Comment by harens** [comment]
Hi there @Tropical-Smootie  👋 

Hope you're well (and great name btw). The error is happening because you installed it with python 3.9, which ciphey doesn't currently support.

```
File "/usr/local/Cellar/python@3.9/3.9.0_1/...
```

The current workaround is to use Docker (see the [wiki](https://github.com/Ciphey/Ciphey/wiki/Common-Issues-&-Their-Solutions#i-cant-install-ciphey-on-mac-os) for more info). Alternatively, I recently wrote a portfile for [MacPorts](https://www.macports.org) that fixes this issue (see https://github.com/Ciphey/Ciphey/issues/583)

```
sudo port install ciphey
```

However, I see you have Homebrew installed, and so although you can have both Homebrew and MacPorts installed (I do ✋), I wouldn't necessarily recommend it.

I therefore suggest you try the Docker method. Let me know if you have any questions about this and I'd be happy to help.

```
docker run -it --rm remnux/ciphey
```

**Reply by bee-san** [reply]
Thanks for this Haren!
